### ISSUE TITLE
Polly: `SynthesizeSpeechInput` doesn't have `Engine` field

### ISSUE THREAD

**Issue by abdul-rehman0** [aws/aws-sdk-go-v2#353]
### Version of AWS SDK for Go?
v0.10.0

### Version of Go (`go version`)?
go1.12.7 linux/amd64

### What issue did you see?
`Engine` field for `SynthesizeSpeechInput` is missing in `aws-sdk-go-v2`. 

In `aws-sdk-go` it is there:
```
type SynthesizeSpeechInput struct {
    // Specifies the engine (standard or neural) for Amazon Polly to use when processing
    // input text for speech synthesis. Using a voice that is not supported for
    // the engine selected will result in an error.
    Engine *string `type:"string" enum:"Engine"`
```

**Comment by sipvoip** [comment]
I also need this, neural has been out for a while, when will the SDK be updated?

**Comment by abdul-rehman0** [comment]
Resolved in v0.11.0.
